### ISSUE TITLE
Fix build issues, update documentation, push new version

### ISSUE THREAD

**Issue by pinkerton** [pinkerton/django-ical#13]
Looks like builds are [failing](https://travis-ci.org/pinkerton/django-ical/jobs/310427987) and the documentation needs being updated for the added calendar fields. 

I will get on this in the coming days and push a new version to PyPi. 🔜

**Comment by coredumperror** [comment]
I would greatly appreciate this! I'm about to start developing iCal functionality into my django site, and a fresh version of this library looks like it would be just the ticket.

**Comment by pinkerton** [comment]
For sure, I should have the new version out by tomorrow, pending any
unforeseen issues! For now, feel free to look at the code for the supported
iCal features and use master in your project.

**Comment by coredumperror** [comment]
Great!

**Comment by pinkerton** [comment]
Hey, sorry to keep you waiting on this. I realized when updating the documentation for this that recurring events are not completely implemented (at least to my knowledge). This is specifically related to https://github.com/pinkerton/django-ical/pull/8

I need to do some testing with this before I'm comfortable pushing out a new version containing those features. Is there anything you need immediately? Apologies to keep you waiting on a release - I am not actively doing Djano development, so it will take some time to get the context to resolve this.

**Comment by coredumperror** [comment]
No worries. The version I'm pulling from the master branch seems to fill my needs just fine, and I'm not in any particular hurry to get a more official version. A few days' won't be a problem.

**Comment by coredumperror** [comment]
Any progress on this? I've got my implementation all up and running, so it'd be nice to have a new official PyPI build to use.

**Comment by pinkerton** [comment]
Hey, yes I will have a new version pushed to PyPI very soon! Just got back from holiday. Apologies!

**Comment by WakeRealityDev** [comment]
Is this updated to work with Django 2.0.1? Thanks

**Comment by coredumperror** [comment]
Hate to be a bother, but I just noticed the note I left in my pip requirements file about django-ical, reminding me to get the new PyPI build. But it's still not out. :(

Is there anything I can do to help get the code ready for a new release?

**Comment by pinkerton** [comment]
Hey, really sorry about this. I've had to prioritize university and consulting work recently over open source. I definitely need to find a maintainer who actively works with Django. If that is you, then happy to chat!

I can push out the new build trivially, but I'm worried I merged a PR that included only half-baked functionality for something. Will get to the bottom of this and either revert the offending code or finish implementing it and push the new version.

**Comment by coredumperror** [comment]
I would actually be uniquely capable as a co-maintainer, since I've been maintaining a Drupal module that does much the same thing as django-ical. However, I'm not sure I could really dedicate myself to working on this module to any meaningful degree.

I'll look forward to a new version, whenever you get the time available to get it built.

**Comment by mbaechtold** [comment]
@pinkerton, there is a project called [Jazzband](https://jazzband.co/), a community taking care of Django addons seeking new maintainers. It already takes care of some of the more popular addons like [django-debug-toolbar](https://github.com/jazzband/django-debug-toolbar), [django-model-utils](https://github.com/jazzband/django-model-utils), [django-axes](https://github.com/jazzband/django-axes) and more.

I can only recommend passing maintainership over to Jazzband.

**Comment by mbaechtold** [comment]
By the way, the tests are failing because `icalendar` >= 4.0 is no longer compatible with Python 2.6.

I see two options here:
- `django-ical` must use `icalendar` < 4.0 in order to support Python 2.6
- `django-ical` could drop support for Python 2.6 too

I prefer the second option. I will submit a pull request if you agree to it.

**Comment by coredumperror** [comment]
+1 on the "drop Python 2.6 support". Pip is dropping support for 2.6, and you can't even use Django 1.7+ without being on at least Python 2.7.

**Comment by pinkerton** [comment]
I'll go ahead and fix that, thanks for pointing it out!

And I'll see about turning over maintainership - that's a great idea.

**Comment by gabn88** [comment]
It would be great to keep this module maintained (up to Django 2.0). Any news regarding passing ownership or updating the module?

**Comment by bhavik2491** [comment]
Any update on new release ? As we're expecting new release since long time as it has been 2 years now.
Kindly put your comment.

**Comment by pinkerton** [comment]
I'm in the process of transferring ownership of the project over to Jazzband. Hopefully their community will be able to give this project the love it deserves ❤️

**Comment by pinkerton** [comment]
Resolved in #19 

Will push the new release tonight and transfer project ownership.

**Comment by bhavik2491** [comment]
Thanks a lot

**Comment by coredumperror** [comment]
This is great! So glad a new version is coming.
